proposed/clock-meta.md

```diff
@@ -64,25 +82,6 @@ final class SystemClock implements \Psr\Clock\ClockInterface
     }
 }
 
-//
-
-final class UTCClock implements \Psr\Clock\ClockInterface
-{
-    private \DateTimeZone $utcTimeZone;
-
-    public function __construct()
-    {
-        $this->utcTimeZone = new \DateTimeZone('UTC');
-    }
-
-    public function now(): \DateTimeImmutable
-    {
-        return new \DateTimeImmutable('now', $this->utcTimeZone);
-    }
-}
```

Comment:
Why was this removed?

Reply:
If we are proposing that the timezone is not part of the Interface we should not provide a minimum implementation that provides a timezone specific implementation. And for a minimum implementation the UTCClock is not necessary. If people want a more sophisticated example they can check out one of the multiple provided implementations